spec/fortnox/api/repositories/examples/only.rb

```diff
@@ -0,0 +1,42 @@
+
+shared_examples_for '.only' do |matching_filter, missing_filter|
+  describe '.only' do
+
+    let( :repository ){ described_class.new }
+    let( :vcr_dir ){ repository.options.json_collection_wrapper.downcase }
```

Comment:
I meant that you fetch the `vcr_dir` from `repository.options.json_collection_wrapper.downcase` :)